ci/licenses_golden/licenses_third_party

```diff
@@ -63661,4 +63661,32 @@ freely, subject to the following restrictions:
 3. This notice may not be removed or altered from any source distribution.
 ====================================================================================================
 
-Total license count: 867
+====================================================================================================
+LIBRARY: react-native
+ORIGIN: ../../../third_party/react-native/LICENSE
+TYPE: LicenseType.mit
+FILE: ../../../third_party/react-native/Libraries/Animated/animations/SpringAnimation.js
+----------------------------------------------------------------------------------------------------MIT License
```

Comment:
maybe just `----------------------------------------------------------------------------------------------------` ? 
Why there is a `MIT License` trailing? I didn't find this trailing in others.... 🤔

Reply:
ah, that might be it, just pushed...we'll see!